command/state_meta.go

```diff
@@ -23,8 +25,9 @@ func (c *StateMeta) State(m *Meta) (state.State, error) {
 		return nil, err
 	}
 
+	env := c.Env()
```

Comment:
I finally see what the second Meta is for now. I'm going to split the Meta back out just to alleviate the confusion, since `StateMeta` doesn't it need.